code/index.html

```diff
@@ -1,30 +1,324 @@
-
-    <!-- Signup form -->
-
-    <section class="accordion">
-      <!-- Accordion -->
-      <h1>FAQ</h1>
-
-      <div class="question" id="section1">Section 1</div>
-      <div class="answer">
-        <p>Lorem ipsum...</p>
+
+<head>
+  <meta charset="utf-8" />
+  <!-- Viewport fit: cover is for iphone overflow -->
+  <meta name="viewport" content="width=device-width, initial-scale=1.0, viewport-fit=cover" />
+  <title>DEATH BY SPAM</title>
+  <!-- Favicon -->
+  <link rel="apple-touch-icon" sizes="180x180" href="./apple-touch-icon.png">
+  <link rel="icon" type="image/png" sizes="32x32" href="./favicon-32x32.png">
+  <link rel="icon" type="image/png" sizes="16x16" href="./favicon-16x16.png">
+  <link rel="manifest" href="/site.webmanifest">
+  <!-- Link Google Fonts -->
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@100;200;300;400;500;600;700;800;900&family=Roboto+Mono:wght@100;200;300;400;500;600;700&family=Roboto:wght@100;300;400;500;700;900&display=swap" rel="stylesheet">
+  <!-- Link Stylesheet -->
+  <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+
+  <!-- Header Section -->
+  <header class="header">
+
+    <!-- Navbar / Menu -->
+    <div class="menu">
+      <img class="menu__logo menu__logo--hidden" src="images/hacking.png" alt="brand logo" />
+      <a class="menu__link" href="#faq" alt="F.A.Q.">
+        FAQ
+      </a>
+      <img class="menu__logo" src="images/hacking.png" alt="brand logo" />
+      <a class="menu__link" href="#signup" alt="Do it!">
+        DO IT!
+      </a>
+    </div>
+
+    <!-- Brand Container over image -->
+    <div class="brand-container">
+      <h1 class="brand brand-container__brand">
+        DEATH BY SPAM
+      </h1>
+    </div>
+  </header>
+
+  <!-- Page -->
+  <section class="page">
+
+    <!-- Intro Text -->
+    <div class="intro">
+      <p class="intro__text">
+        Got a particularly hateful co-worker? An annoying ex? Shitty parents (who suck at tech)? Perhaps a mortal enemy?
+        <br><br>
+        Use our service to sign them up for up to 400+ newsletters and over 50 telemarketing companies.
+        <br><br>
+        All in less than two minutes. 😈
+      </p>
+    </div>
+
+    <!-- FAQ Accordion -->
+    <div class="faq" id="faq">
+      <h1 class="title faq__title">
+        FREQUENTLY ASKED QUESTIONS
+      </h1>
+
+      <button class="subtitle faq__question" id="question1">
+        How does this work?
+      </button>
+      <div class="faq__answer">
+        <p class="text faq__text">
+          Fill out your victim's information in the <strong>R.I.P. INBOX</strong> form. After selecting the various options that might "interest" them, our nifty bot signs them up for newsletters and mailing lists.
+          <br><br>Click <strong>SUBMIT</strong>. Violence achieved.
```

Comment:
In below "Common Pitfall" I read we shoud avoid <br> and stick to <p> as default display: block, but Im often tempted to use it, and think its difficult living withouth it:-)
(https://www.notion.so/Week-2-common-pitfalls-bbe695289d154ff98d6a247cef873f07)

Reply:
Ooh I should read up on that again. But I agree - `display:block` is a little hard to live without 😭